lib/fortnox/api/types.rb

```diff
@@ -38,6 +38,15 @@ module Types
       AccountNumber = Strict::Int
                       .constrained(gteq: 0, lteq: 9999)
                       .optional
+                      .constructor do |input|
+                        unless input.nil? || input.to_s.empty?
+                          Integer(input)
+                        else
+                          input
+                        end
+                      rescue ArgumentError
+                        input
```

Comment:
Is this case tested?

Reply:
Not all valid/invalid cases are tested. We could test for everything but right now, with my suggested changes, we do test the two main cases: Things that respond to to_s and has a valid value will convert, the rest will cause a Dry types constraint error...